map/src/test/java/openlr/map/utils/GeometryUtilsTest.java

```diff
@@ -480,6 +491,12 @@ public final void testCoordinateBounds() {
                 Double.MIN_VALUE));
     }
 
+    @Test
+    public void bearingDifferenceTest() {
```

Comment:
A couple of more tests where the first angle isn't 0 would be good.  eg.

```java
assertEquals(GeometryUtils.bearingDifference(345, 10), 25);
assertEquals(GeometryUtils.bearingDifference(150, 220), 70);
```

Reply:
458ee63d9b353f59b5145a2bb129eef2fe2ab6ad

Reply:
78af58f0f964ce6e2c59d398baa21759cfa62a88